error: 'GrbLogger' does not name a type

Last successful build was right before june 13th; so I guess the following commit did something?
https://github.com/gerbera/gerbera/commit/fb914d9b55a6751a3b97a7e9d59eccf83e23f9f7


[ 78%] Building CXX object CMakeFiles/libgerbera.dir/src/util/logger.cc.o
/opt/_Build/gerbera-git/src/util/logger.cc:131:1: error: ‘GrbLogger’ does not name a type
  131 | GrbLogger GrbLogger::Logger;
      | ^~~~~~~~~
make[2]: *** [CMakeFiles/libgerbera.dir/build.make:1644: CMakeFiles/libgerbera.dir/src/util/logger.cc.o] Fehler 1
make[2]: *** Auf noch nicht beendete Prozesse wird gewartet …
make[1]: *** [CMakeFiles/Makefile2:85: CMakeFiles/libgerbera.dir/all] Fehler 2
make: *** [Makefile:156: all] Fehler 2

--- a/src/util/logger.cc
+++ b/src/util/logger.cc
@@ -23,9 +23,9 @@
 
 /// \file logger.cc
 
-#ifdef GRBDEBUG
-
 #include "util/logger.h" // API
+
+#ifdef GRBDEBUG
 
 #include <numeric>
 

